cmd/operator-sdk/new/cmd.go

```diff
@@ -303,17 +241,12 @@ func doAnsibleScaffold() error {
 }
 
 func verifyFlags() error {
-	if operatorType != projutil.OperatorTypeAnsible && operatorType != projutil.OperatorTypeHelm {
-		return fmt.Errorf("value of --type can only be `ansible`, or `helm`: %v",
-			projutil.ErrUnknownOperatorType{Type: operatorType})
-	}
-	if operatorType != projutil.OperatorTypeAnsible && generatePlaybook {
+	if generatePlaybook {
 		return fmt.Errorf("value of --generate-playbook can only be used with --type `ansible`")
 	}
```

Comment:
This will always result in an error when `generatePlaybook` is set, which is not what we want. Remove this check and the error that's returned.

Reply:
In fact, it looks like this function is now the same as `apiFlags.VerifyCommonFlags()`, so remove this function entirely and call `apiFlags.VerifyCommonFlags()` directly.